lib/src/ffmpeg/filters/audio_mix_filter.dart

```diff
@@ -2,14 +2,15 @@ import 'package:ffmpeg_cli/src/ffmpeg/ffmpeg_command.dart';
 
 /// Mixes multiple audio streams together into one.
 ///
-/// I think this will automatically cut the volume of each input proportional
+/// This will automatically cut the volume of each input proportional
 /// to the number of inputs, e.g., 1/2 volume for 2 inputs, 1/3 volume for 3
-/// inputs.
+/// inputs. (1/n)
 class AMixFilter implements Filter {
   const AMixFilter({
     required this.inputCount,
   });
 
+  /// Number of inputs (defaults to 2)
```

Comment:
I don't see a default

Reply:
Changed so that the inputCount can be nullable so that it reflects the ffmpeg documentation where if the argument is omitted, it will use the ffmpeg default value of 2 as described [here](https://ffmpeg.org/ffmpeg-filters.html#amix).